### PR TITLE
Added basic monitoring solution with Prometheus & Grafana

### PR DESCRIPTION
**Pull request by noamgu**

### CODE REVIEW

File: bootstrap-monitoring.sh [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+# Delete unused files
+rm -rf grafana*
+
+# Place config files & dashbord
+cp -rf /vagrant/monitoring_assets/datasource.yml /etc/grafana/provisioning/datasources/
+cp -rf /vagrant/monitoring_assets/dashboard.yml /etc/grafana/provisioning/dashboards
+cp -rf /vagrant/monitoring_assets/devopsloft.json /etc/grafana/provisioning/dashboards/
+
+# Start & Enable grafana service
+service grafana-server start
```
Comment (lioramilbaum) [inline bootstrap-monitoring.sh]: starting a service with one method and enabling it with another method (systemd) is not the best practice

File: README.md [hunk 2]
```diff
@@ -104,3 +104,23 @@ vagrant up stage
 ```
 vagrant destroy --force stage
 ```
+
+## Monitoring
+
+* Prometheus
+```
+# Dashboard
+localhost:9090
```
Comment (lioramilbaum) [inline README.md]: this is true for dev environment. what about stage/prod?
what about verifying the ports are open in stage/prod?

**Comment by Balex3000** [comment]
Tested on both dev and stage. 
Ports for stage were not needed in Vagrantfile but they have to be opened in security group on AWS

File: README.md (hunk 2 above)
Comment (noamgu) [reply README.md]: Updated the Readme for @Balex3000 Doc.
Also tested in stage ENV by Alex.

**Pull request closed without merging**